katsdpingest/ingest_session.py

```diff
@@ -8,9 +8,12 @@
 import argparse
 import textwrap
 import gc
+import urllib.parse
 from typing import Mapping, Dict, List, Tuple, Set, Iterable, Optional, Any   # noqa: F401
 
 import numpy as np
+import astropy.units as u
```

Comment:
minor, but personally I would prefer to leave the full package name since u. is only used twice and it makes the code more readable to have astropy.units. instead.

Reply:
Calling this `u` seems to be a convention, similar to how everyone uses `np` for numpy.